ceph_installer/util.py

```diff
@@ -38,6 +38,22 @@ def generate_inventory_file(inventory, task_uuid, tmp_dir=None):
     return inventory_file.name
 
 
+def validate_monitors(monitors, host):
+    """
+    Ensures that the given host is not included in
+    the monitors list. This fixes an issue with users
+    trying to provide the monitor they are currently
+    configuring in the list of 'monitors'.
+    """
+    result = []
+    for mon in monitors:
+        if mon["host"] == host:
+            logger.info("%s removed from list of monitors", host)
```

Comment:
this should be a `logger.warning` along with the "why". Two log lines would be OK if one is too much